doc/source/programs/gdal_raster_clip.rst

```diff
@@ -60,19 +60,72 @@ Standard options
     Instead, the bounds are reprojected from the bbox-crs to the CRS of the
     input dataset.
 
+.. option:: --geometry <WKT_or_GeoJSON>
+
+    Geometry as a WKT or GeoJSON string to which to clip the dataset.
+    Raster areas within the bounding box of the geometry but not inside the
+    geometry itself will be set to the nodata value of the raster, or 0 if there
+    is none.
+    It is assumed to be in the CRS of
```

Comment:
Implementation assumes input geometry in GeoJSON is WGS84. (I think the behavior documented here would be less surprising, FWIW)

Reply:
changed to "If the input geometry is GeoJSON, its CRS is assumed to be WGS84, unless there is
    a CRS defined in the GeoJSON geometry or :option:`--geometry-crs` is specified.
    If the input geometry is WKT, its CRS is assumed to be the one of the input dataset,
    unless :option:`--geometry-crs` is specified."